backend/src/api/organizations.ts

```diff
@@ -147,6 +155,56 @@ export const get = wrapHandler(async (event) => {
   };
 });
 
+class UpdateBody {
+  @IsBoolean()
+  enabled: boolean;
+}
+
+export const updateScan = wrapHandler(async (event) => {
+  const organizationId = event.pathParameters?.organizationId;
+
+  if (!organizationId || !isUUID(organizationId)) {
+    return NotFound;
+  }
+
+  if (!isOrgAdmin(event, organizationId)) return Unauthorized;
+
+  await connectToDatabase();
+  const scanId = event.pathParameters?.scanId;
+  if (!scanId || !isUUID(scanId)) {
+    return NotFound;
+  }
+  const scan = await Scan.findOne(
+    {
+      id: scanId
+    },
+    {
+      relations: ['organizations']
+    }
+  );
+  if (scan) {
+    const body = await validateBody(UpdateBody, event.body);
+    const existing = scan.organizations.find(
+      (org) => org.id === organizationId
+    );
+    if (body.enabled && !existing) {
+      scan.organizations.push(
+        plainToClass(Organization, { id: organizationId })
+      );
+    } else if (!body.enabled && existing) {
+      scan.organizations = scan.organizations.filter(
```

Comment:
I'm worried about cases where two different people call this function here. Probably better to modify the organization model instead of the scan model.